framework/src/st_things/things_stack/framework/things_req_handler.c

```diff
@@ -190,6 +190,96 @@ OCEntityHandlerResult send_response(OCRequestHandle request_handle, OCResourceHa
 	return eh_result;
 }
 
+static OCEntityHandlerResult convert_ap_infor(things_resource_s *target_resource, access_point_info_s *p_list, int list_cnt)
+{
+	things_representation_s *rep = NULL;
+
+	if (list_cnt == 0 || p_list == NULL) {
+		THINGS_LOG_E(TAG, "Scaned AP list is NULL ");
+		return OC_EH_ERROR;
+	}
+
+	if (target_resource->rep == NULL) {
+		rep = things_create_representation_inst(NULL);
+		target_resource->things_set_representation(target_resource, rep);
+	} else {
+		rep = target_resource->rep;
+	}
+	things_representation_s **child_rep = (things_representation_s **)things_malloc(list_cnt * sizeof(things_representation_s *));
+	access_point_info_s *wifi_scan_iter = p_list;
+	int iter;
+	for (iter = 0; iter < list_cnt; ++iter) {
```

Comment:
How about unifying with "iter++" or "++iter"?

Reply:
In this case ( for loop)  both iter++ and ++iter give the same result. But ++iter give  more efficient result than iter++

Reply:
Ok then apply line 230 also

Reply:
ok.

Reply:
fixed.